README.md

```diff
@@ -359,6 +359,31 @@ View Kubescape scan results directly in [Lens IDE](https://k8slens.dev/) using k
 
 </details>
 
+
+## Build on pre-configured killercoda's ubuntu playground
+
+* [Pre-configured Killercoda's Ubuntu Playground](https://killercoda.com/suhas-gumma/scenario/kubescape-build-for-development)
+
+<details><summary> Pre-programmed actions executed by the playground </summary>
+
+
+* Clone the official GitHub repository of `Kubescape`.
+* Install cmake.
+* Install `libgit2` dependency inside 'kubescape' directory.
+* Execute `make build` inside 'kubescape' directory.
+* The entire process involves executing multiple commands in order and it takes around 5-6 minutes to execute them all.
+
+</details>
+
+<details>
+<summary>Instructions to use the playground</summary>
+
+* Apply changes you wish to make to the kubescape directory using text editors like `Vim`.
+* Execute `make build` after the changes. (This will build the `kubescape` binary)
```

Comment:
Here again, we should have a [single place](https://github.com/kubescape/kubescape#build-on-linuxmacos) of instructions on how to build